r-api-management.tf

```
@@ -30,7 +30,8 @@ resource "azurerm_api_management" "apim" {
   }
 
   identity {
-    type = var.identity_type
+    type         = var.identity_type
+    identity_ids = var.identity_ids
```

Does this works well in case of "System Assigned" identity ? I think `null` is expected instead of `[]`.

```suggestion
    identity_ids = replace(var.identity_ids, "User Assigned", "") == var.identity_ids ? null : var.identity_ids
```